resources/db/migration_mariadb/V6_1__indirect_upgrade.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE indirect_incoming ADD amount bigint(20) DEFAULT '0';
```

Comment:
what's the difference between bigint(20) and BigInt (like in V6_1_indirect_upgrade.sql) - could this cause inconsistencies?